client/extensions/woocommerce/app/store-stats/referrers/index.js

```diff
@@ -57,11 +59,22 @@ class Referrers extends Component {
 	}
 
 	onSearch = str => {
+		if ( str === '' ) {
```

Comment:
Should we `trim()` the search input to handle empty spaces? I'm okay with this being in a follow-up branch